.github/workflows/lint.yml

```
@@ -25,7 +25,7 @@ jobs:
         run: bundle exec rubocop
 
       - name: Run Slim-Lint with PR annotations
```

From the change below looks like we're removing "report with PR annotations" flag. 
Adapting the description:
```suggestion
      - name: Run Slim-Lint
```